### PR TITLE
NPC unaggro timer: Added a notification feature to the Unaggro plugin

### PR DESCRIPTION
**Pull request by h0dgep0dge**
Not necessarily looking to have this merged as-is, feedback is welcome

### CODE REVIEW

**Comment by h0dgep0dge** [comment]
Thanks, I wasn't aware that there were additional steps to getting the code merged. I'd expected something along the lines of an interrogation and defense lol sorry if there's a prescribed procedure I've missed somewhere

**Comment by Nightfirecat** [comment]
No need for that. Good code and a good use case is enough to get code merged most of the time.

File: runelite-client/src/main/java/net/runelite/client/plugins/npcunaggroarea/NpcAggroAreaPlugin.java [hunk 1]
```diff
@@ -125,6 +129,9 @@
 	private WorldPoint lastPlayerLocation;
 	private WorldPoint previousUnknownCenter;
 	private boolean loggingIn;
+
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/npcunaggroarea/NpcAggroAreaPlugin.java]: Remove this extra newline.
```suggestion
```

Comment (Nightfirecat) [reply runelite-client/src/main/java/net/runelite/client/plugins/npcunaggroarea/NpcAggroAreaPlugin.java]: Also, this should be reset along with the other plugin fields in `shutDown()`.

**Pull request merged**